src/System.Numerics.Tensors/System/Numerics/Tensor.cs

```diff
@@ -1431,6 +1430,54 @@ public virtual CompressedSparseTensor<T> ToCompressedSparseTensor()
 
         #endregion
 
+        #region Slice
+
+        public Tensor<T> Slice(params Range[] ranges)
+        {
+            return Slice(new ReadOnlySpan<Range>(ranges));
+        }
+
+        public Tensor<T> Slice(ReadOnlySpan<Range> ranges)
+        {
+            if (ranges.Length != Rank)
+            {
+                throw new ArgumentException($"{nameof(ranges.Length)} of {nameof(ranges)} ({ranges.Length}) must match the {nameof(Rank)} of this {nameof(Tensor)} ({Rank}).");
+            }
+
+            Span<int> newDimensions = Rank < ArrayUtilities.StackallocMax ? stackalloc int[Rank] : new Span<int>(new int[Rank]);
```

Comment:
You should be able to rewrite this as follows (using implicit cast):
```C#
Span<int> newDimensions = Rank < ArrayUtilities.StackallocMax ? stackalloc int[Rank] : new int[Rank];
```

Reply:
Fixed in all places I could find.